README.md

```diff
@@ -9,9 +9,15 @@ More information is available at https://community.ubuntu.com/t/faq-ubuntu-new-t
 
 Detailed screenshots and reasoning behind the design can be found inside the Design Wiki: https://github.com/ubuntu/yaru/wiki
 
-![Files](https://i.imgur.com/XA00uS9.jpg)
+![Files](https://i.imgur.com/Qxl69M6.png)
```

Comment:
can we use the official wallpaper instead?

Reply:
That is actually one of the wallpapers installed by default :) have a look

Reply:
Yes,  but I meant the one you have soon after install :)

Reply:
done